### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by 5443322** [kubilus1/autoortho#480] [General]
![01_pad20210428T110317__esmma6b](https://github.com/kubilus1/autoortho/assets/147063594/56377ef4-a476-4074-b1ef-9a28fa8b3bc5)
![Screenshot 2024-01-20 112022](https://github.com/kubilus1/autoortho/assets/147063594/87c8cb77-c9b8-4132-91c1-84198b337be5)

How can I fix it? URKG

**Comment by Jeff2122** [comment]
My understanding is that AutoOrtho has nothing to do with the un-level terrain. That is all in the airport design via Scenery Gateway, and is within XP12. Search the forums on XP.org and you can see others showing similar issues. Ortho is simply a mask over the land, the mesh is the terrain. 
That being said, I don't have a better explanation as to why your terrain is funky with AO. But I learn a bit more each day.
